.travis.yml

```diff
@@ -1,5 +1,7 @@
 sudo: false
 
+dist: trusty
```

Comment:
the default distro for travis is xenial, were there problems using that one?

Reply:
Xenial does not support Java 8 any more...

Reply:
I think if you use `openjdk8` instead of `oraclejdk8` it should work